agents-common/src/main/java/org/apache/ranger/admin/client/datatype/GrantRevokeData.java

```diff
@@ -25,10 +25,10 @@
 
 import org.apache.ranger.authorization.utils.StringUtil;
 import org.apache.ranger.plugin.util.JsonUtilsV2;
-import org.codehaus.jackson.annotate.JsonAutoDetect;
-import org.codehaus.jackson.annotate.JsonAutoDetect.Visibility;
-import org.codehaus.jackson.annotate.JsonIgnoreProperties;
-import org.codehaus.jackson.map.annotate.JsonSerialize;
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 
 
 @JsonAutoDetect(getterVisibility = Visibility.NONE, setterVisibility = Visibility.NONE, fieldVisibility = Visibility.ANY)
```

Comment:
With the move to fasterxml.jackson 2.x, consider replacing deprecated `include = JsonSerialize.Inclusion.NON_EMPTY`, with `@JsonInclude(JsonInclude.Include.NON_EMPTY)`

Reply:
As stated above, the changes have been implemented with https://github.com/apache/ranger/commit/cccbabf969cc1f349c91d2c6de424380b0a8c49a commit.